DESIGN_GUIDELINES.md

```diff
@@ -56,6 +116,69 @@ allows the library to be used from all supported programming languages. jsii
 poses restrictions on language features that cannot be idiomatically represented
 in target languages.
 
+## What's Included
+
+The AWS CDK includes the AWS Construct Library, which contains constructs
+representing AWS resources.
+
+The AWS Construct Library has three different levels of constructs, beginning
+with low-level constructs, which we call _CFN Resources_ (or L1, short for
+"level 1") or Cfn (short for CloudFormation) resources. These constructs
+directly represent all resources available in AWS CloudFormation. CFN Resources
+are periodically generated from the AWS CloudFormation Resource
+Specification. They are named **Cfn**_Xyz_, where _Xyz_ is name of the
+resource. For example, CfnBucket represents the AWS::S3::Bucket AWS
+CloudFormation resource. When you use Cfn resources, you must explicitly
+configure all resource properties, which requires a complete understanding of
+the details of the underlying AWS CloudFormation resource model.
+
+The next level of constructs, L2, also represent AWS resources, but with a
```

Comment:
Would it be worth mentioning that L2 resources interact with other CloudFormation resources only via other L2s, never via L1s?

Reply:
Definitely worth capturing somewhere in this doc, but I'm not sure if this is the right section. Maybe under the `Props` section, or `Implementation`, depending on if we're talking about the public surface area or implementation details?